keyboards/handwired/dactyl/base/promicro/keymaps/default/keymap.c

```diff
@@ -1,9 +1,11 @@
 #include QMK_KEYBOARD_H
 
 
-#define _QWERTY 0
-#define _LOWER 1
-#define _RAISE 2
+enum layers {
```

Comment:
This isn't correct.  Layers should start at 0 (which doesn't need definition)

Reply:
I'm not following, are you asking to revert from `enum layers` to three `#define`, or remove `= SAFE_RANGE`?

Reply:
You need to remove `= SAFE_RANGE`, otherwise the first layer will be `~32.000` instead of `0`, which will cause several errors

Reply:
fix applied/issue addressed